azure-pipelines/prep-release.yml

```diff
@@ -0,0 +1,39 @@
+trigger:
+  - none
+
+pr:
+  - none
+
+schedules:
+  - cron: "00 5 * * *"
```

Comment:
What will happen if there have been no changes since last release? This would happen on weekends, where we haven't merged anything since the previous night.

Reply:
If there are no news files that would affect the version, the pipeline will fail and won't create a release.

Reply:
Yes, I suspected the pipeline would fail. I guess we need to be okay with failures of the pipeline, or add some logic to detect change and exit the pipeline with no error. It seems best to be that failure of the pipeline should be reserved for when the pipeline isn't working as intended, rather than being nominal, so we can use failure to prompt ourselves to go fix the pipeline. The failure signal is going to be less meaningful (and perhaps mask real failures) if it is accepted as normal.

Reply:
Changed the script to exit early when the version can't be bumped instead of failing.